### ISSUE TITLE
ID3 Title

### ISSUE THREAD

**Issue by siam28** [mlj/castget#3]
Hello,

Is there anyway to set the ID3 tag title to the <title> ... </title> enclosure of the retrieved item by default?

Thanks

**Comment by mlj** [comment]
No, there isn't, only the static values from the configuration file are supported. But this is certainly a useful enhancement and goes on the TODO list. Feel free to have a go at implementing it if you want to :)

**Comment by LennyM8472** [comment]
Regarding id3 tags.

My Config doesn't seem to write any of the id3 features into the dowloaded mp3 files.

Any ideas why it doesn't work?

[CLR]
url=http://www.cl-rec.com/pod/podcast
spool=/share/Multimedia/Podcast/CLR/
id3leadartist=CLR AM/FM Podcast
id3album=CLR AM/FM Podcast
playlist=/share/Multimedia/Podcast/CLR.m3u

best regards
Lennart

**Comment by mlj** [comment]
Works for me:
```
$ id3info AM_FM042_Chris_Liebing.mp3

*** Tag information for AM_FM042_Chris_Liebing.mp3
=== TSSE (Software/Hardware and settings used for encoding): Logic 10.0.7
=== COMM (Comments): (iTunNORM)[eng]:  00000593 00000628 00014795 0001F1E1 002CF8C1 002C7278 00009105 00009093 0014C4BC 00149478
=== COMM (Comments): (iTunSMPB)[eng]:  00000000 00000210 00000930 0000000009767C40 00000000 05831EB8 00000000 00000000 00000000 00000000 00000000 00000000
=== APIC (Attached picture): (����=Exif)[, 0]: JPG, 1316752 bytes
=== TPE1 (Lead performer(s)/Soloist(s)): CLR AM/FM Podcast
=== TALB (Album/Movie/Show title): CLR AM/FM Podcast
=== TCON (Content type): Podcast
=== COMM (Comments): (ID3v1 Comment)[XXX]:  00000593 00000628 00014795 00
*** mp3 info
MPEG1/layer III
Bitrate: 64KBps
Frequency: 44KHz
```

Are you sure you have ID3-support compiled in? You can check with something like this:

```shell
$ ldd /usr/bin/castget | grep libid3
libid3-3.8.so.3 => /usr/lib/libid3-3.8.so.3 (0x00007f9453cf7000)
```

**Comment by LennyM8472** [comment]
Hi,

i have installed libid3 or in my case id3lib in the system. Both installed via the same  entware environment on a Qnap device.

[~] # opkg install id3lib
Package id3lib (3.8.3-2) installed in root is up to date.

but I think the castget application is not able to access id3lib

Lenny

**Comment by mlj** [comment]
I'm not familiar with the entware environment, so I may be looking in the wrong place, but it appears ID3 support is explicitly disabled in the package (see https://github.com/Entware/rtndev/blob/master/castget/Makefile#L42).

You should probably get in touch with the entware team about this.
